test/unit/spec/common/jobCommChannel-spec.js

```diff
@@ -13,15 +13,10 @@ define([
     // allow spies to be overwritten
     jasmine.getEnv().allowRespy(true);
 
-    const TEST_JOB_ID = 'someJob',
+    const { TEST_JOB_ID } = JobsData,
```

Comment:
All the changes in this file are variable name substitutions to make this file consistent with the other tests.